blockchain/error.go

```diff
@@ -417,6 +417,10 @@ const (
 	// ErrInvalidEarlyVoteBits indicates that a block before stake validation
 	// height had an unallowed vote bits value.
 	ErrInvalidEarlyVoteBits
+
+	// ErrVersionVoting indicates that there was an error during version
+	// voting accounting.
+	ErrVersionVoting
```

Comment:
Nothing uses this, so it should be removed in this PR.  Also, `RuleError` is intended specifically for consensus errors.  Thus, an internal error which prevented counting the votes due to a bug in the code should definitely not be signaled as a `RuleError` anyways.

Once the time comes that the version is actually enforced, there should be an additional error added here that mirrors `ErrBlockVersionTooOld`, but that should be part of the PR that adds enforcement.